app/views/ownership_certificates/show.html.erb

```diff
@@ -22,7 +22,7 @@
         </p>
         <p class="govuk-body">
           If you know the other owners' details, you must tell them about this application. You need to do this in writing
-          (by email or letter). You can use this notification form to share the right information
+          (by email or letter). You can use this <%= link_to "notification form", "https://ecab.planningportal.co.uk/uploads/1app/notices/notice1.pdf", class: "govuk-link" %> to share the right information.
```

Comment:
should this have `target='_blank'`?

Reply:
Yes  🤦‍♀️ - thanks, will push an update now!